pkg/cmd/workspace/create.go

```diff
@@ -160,6 +160,7 @@ var CreateCmd = &cobra.Command{
 		}
 
 		stopLogs = true
+		showCursor()
```

Comment:
Remove the showCurosr function - replace with just fmt.Print

Add a comment above: "Make sure terminal cursor is reset"

Reply:
Done!